packages/iTwinUI-react/src/core/Tile/Tile.tsx

```diff
@@ -141,6 +160,29 @@ export const Tile = (props: TileProps) => {
   const showMenu = React.useCallback(() => setIsMenuVisible(true), []);
   const hideMenu = React.useCallback(() => setIsMenuVisible(false), []);
 
+  const renderTitleIcon = () => {
+    if (isLoading) {
+      return (
+        <ProgressRadial
+          className='iui-tile-status-icon'
+          aria-hidden
+          indeterminate
+        />
+      );
+    }
+    if (isSelected) {
+      return <SvgCheckmark className='iui-tile-status-icon' aria-hidden />;
+    }
+    if (isNew) {
+      return <SvgNew className='iui-tile-status-icon' aria-hidden />;
+    }
+    if (!!status) {
+      return StatusIconMap[status]({ className: `iui-tile-status-icon` });
```

Comment:
nit: This can be "JSX-ify'd" if you assign it to a variable (see example in Alert). Makes it easier to work with.

https://github.com/iTwin/iTwinUI-react/blob/7913c3d0b5418d95988d1c74527e422839f542f1/packages/iTwinUI-react/src/core/Alert/Alert.tsx#L71

https://github.com/iTwin/iTwinUI-react/blob/7913c3d0b5418d95988d1c74527e422839f542f1/packages/iTwinUI-react/src/core/Alert/Alert.tsx#L84

Reply:
Using this method.